.github/workflows/react.yml

```diff
@@ -28,6 +28,6 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm install
-    - run: npm run build --if-present
-    - run: npm test
+    # - run: npm install
```

Comment:
As part of this change React and React-native extensions are now built and tested as part of the main ci.yml flow, so once this is checked in we will be able to build, release and test these all together.